dashboard/common/utils.py

```diff
@@ -0,0 +1,13 @@
+import logging
+from decouple import config
+logger = logging.getLogger(__name__)
+
+
+def get_build_info():
+    logger.info(get_build_info.__name__)
+    git_commit=config("OPENSHIFT_BUILD_COMMIT",default="")
+    build_namespace=config("OPENSHIFT_BUILD_NAMESPACE",default="")
+    git_branch=config("OPENSHIFT_BUILD_REFERENCE",default="")
+    build_source=config("OPENSHIFT_BUILD_SOURCE",default="")
+    build_name=config("OPENSHIFT_BUILD_NAME",default="")
+    return 'Build_Namespace:'+build_namespace+' Build_Name:'+build_name+' Git_Source: '+build_source+' Git_Branch:'+git_branch+' Git_Commit:'+git_commit
```

Comment:
This can be written cleaner with fstrings

```suggestion
    return f'Build_Namespace:{build_namespace} Build_Name: {build_name} Git_Source: {build_source} Git_Branch: {git_branch} Git_Commit: {git_commit}'
```

Reply:
Done